pallets/capacity/README.md

```diff
@@ -0,0 +1,59 @@
+# Capacity Pallet
+
+Manages the staking and balances for Capacity, an alternative payment system on Frequency.
+
+## Summary
+
+Capacity is an alternative payment system for a subset of calls that renews each Epoch.
+Tokens staked create Capacity for a targeted Provider.
+[Learn more about Capacity](https://docs.frequency.xyz/Tokenomics/ProviderIncentives.html#capacity-model).
+
+### Staking & Unstaking
+Currently, the token to Capcity ratio is 50:1.
```

Comment:
```suggestion
Currently, the token to Capacity ratio is 50:1.
```

As to @claireolmstead 's capitalization question, I would say that "Capacity" is a specific name unique to Frequency, whereas "token" is a generic blockchain primitive.